Assets/HoloToolkit/Utilities/Scripts/FadeScript.cs.meta

```diff
@@ -1,8 +1,9 @@
 fileFormatVersion: 2
-guid: 794a5a07720daf4479efca14d8ba4297
-timeCreated: 1495751254
+guid: 3227eb50378702c4eba7e61f471ee7c7
```

Comment:
Won't this break existing links and discoverability of the `obsolete` message?

Reply:
No, it'll update correctly.  I tested this in my own project.

Reply:
Oh, you moved the GUID to point to the new script. What's the point of the obsolete marker then, if any existing references will migrate?

Reply:
Just in case someone was referencing the class in code.